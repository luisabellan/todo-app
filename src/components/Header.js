import React from "react";

function Header() {
  return <h2>Todo App</h2>;
}

export default Header;
