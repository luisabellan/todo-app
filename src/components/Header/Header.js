import React from 'react';
import './Header.scss';

function Header() {
  return (
    <div data-testid="header">
      <h2>Todo App</h2>
    </div>
  );
}

export default Header;
