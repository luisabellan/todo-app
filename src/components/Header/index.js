import React from 'react';
import './Header.scss';

function Header() {
  return <h2>Todo App</h2>;
}

export default Header;
