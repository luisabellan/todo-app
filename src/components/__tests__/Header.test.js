import React from 'react';
import '@testing-library/jest-dom';
import userEvent from '@testing-library/user-event';
import { render, screen, cleanup } from '@testing-library/react';
import Header from '../Header';

const todo = {
  id: 1,
  name: 'buy bread',
  completed: false
};

afterEach(cleanup);

describe('todo test', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Header todo={todo} />);
    expect(asFragment()).toMatchSnapshot();
  });

  it('should show title', () => {
    render(<Header />);

    expect(screen.getByText("Todo App")).toBeInTheDocument();
  });
});
