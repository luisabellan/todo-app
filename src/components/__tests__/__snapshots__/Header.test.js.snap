// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`todo test matches snapshot 1`] = `
<DocumentFragment>
  <div
    data-testid="header"
  >
    <h2>
      Todo App
    </h2>
  </div>
</DocumentFragment>
`;
